packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/components/Questions/components/InputGroup/components/QuestionInput.spec.js

```diff
@@ -0,0 +1,259 @@
+import { expect } from 'chai'
+import { Grommet } from 'grommet'
+import React from 'react'
+import sinon from 'sinon'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import zooTheme from '@zooniverse/grommet-theme'
+
+import { QuestionInput } from './QuestionInput'
```

Comment:
This should be importing the decorated `QuestionInput` if you use `Grommet` in the tests.
```suggestion
import QuestionInput from './QuestionInput'
```